cloud/edgecontroller/conf/controller.yaml

```diff
@@ -8,8 +8,12 @@ controller:
     node_update_frequency: 10
     kubeconfig: ""   #Enter path to kubeconfig file to enable https connection to k8s apiserver
 cloudhub:
+  protocol_websocket: true # enable websocket protocol
+  port: 10000 # open port for websocket server
+  protocol_quic: true # enable quic protocol
```

Comment:
if both websocket and quic are true, will cloudHub start both the servers ?  So the edge decides to connect to which server ?

Reply:
yes, cloudhub support both quic and websocket. and edge decides to connect to which server.